spec/operators/multicast-spec.ts

```diff
@@ -548,29 +542,6 @@ describe('Observable.prototype.multicast', () => {
   });
 
   describe('when given a subject', () => {
-    it('should NOT allow you to reconnect by subscribing again', (done: MochaDone) => {
-      const expected = [1, 2, 3, 4];
-      let i = 0;
-
-      const source = Observable.of(1, 2, 3, 4).multicast(new Subject<number>());
-
-      source.subscribe((x: number) => {
-        expect(x).to.equal(expected[i++]);
-      },
-        null,
-        () => {
-          source.subscribe((x: number) => {
-            done(new Error('should not be called'));
-          }, null, () => {
-            done();
-          });
-
-          expect(() => source.connect()).to.throw(Rx.ObjectUnsubscribedError);
-        });
-
-      source.connect();
-    });
-

```

Comment:
Are we reverting the decision we did in https://github.com/ReactiveX/rxjs/issues/453 ?